packages/venia-concept/upward.yml

```diff
@@ -33,11 +33,10 @@ response:
     - matches: request.url.pathname
       pattern: '^/(graphql|rest)(/|$)'
       use: proxy
-    # Requests to create an account are handled by the top-level 'index'
-    # object, which returns a generic app shell
+    # Requests to create an account are handled by the 'appShell' object.
     - matches: request.url.pathname
       pattern: '^/create-account'
-      use: index
+      use: appShell
```

Comment:
Instead of having two routes, `index` vs `appShell` or `hardcodedAppShell` vs `urlResolvedAppShell` as @zetlen and I discussed, I went with this method in which I re-used the single `appShell`. The benefit being that I did not have to rewrite much of the foundation that appShell already provides.